platform-cloud/docs/compute-envs/seqera-compute.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Seqera Compute"
+description: "Instructions to set up Seqera Compute in Seqera Platform"
+date: "21 Apr 2023"
+tags: [seqera compute, compute environment]
+---
+
+Seqera Compute enables you to seamlessly run pipelines and Studios in Seqera Cloud in a fully managed and optimized compute environment on AWS. Seqera automatically provisions and manages all underlying resources, including AWS accounts, credentials, roles, AWS Batch compute environments, and S3 storage buckets, requiring minimal user configuration. Using prepaid credits enables control and visibility of the compute spend in each of your organization's workspaces.
```

Comment:
```suggestion
Seqera Compute enables you to run pipelines and studio sessions in Seqera Cloud in a fully managed and optimized AWS environment. Seqera automatically provisions and manages all the underlying resources, including AWS accounts, credentials, roles, AWS Batch compute environments, and S3 storage buckets, requiring minimal user configuration. Using prepaid credits enables control and visibility of the compute spend in each of your organization's workspaces.
```

Reply:
I think we need to be very careful calling Seqera Compute a "compute environment on AWS". A compute environment is a very specific/defined resource/entity managed by AWS Batch. This is more than a CE. Either just call it an "AWS environment" or we need to think of another descriptive noun that best represents it.